themes/qgrid/layout/partials/nav.ejs

```diff
@@ -1,89 +1,27 @@
 <nav>
-    <h3>Getting started</h3>
-
     <ul>
-        <li>
-            <a href="#">Introduction</a>
-        </li>
-        <li>
-            <a href="#">How to install</a>
-        </li>
-        <li>
-            <a href="#">Concepts and architecture</a>
-        </li>
-        <li>
-            <a href="#">Grid model and service</a>
-        </li>
-    </ul>
+        <% var groupOrder = ['Getting started', 'API', 'Features'];
```

Comment:
consider to have next renamings:

groupOrder -> groups
groupped -> pageGroups